cpp/data/installer/scripts/install_ubuntu_packages.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+cd /
+export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
```

Comment:
Should we also add this to `/etc/profile.d/vufind.sh`? see installer.cc => SetVuFindEnvironmentVariables